package.json

```
@@ -91,7 +91,7 @@
     "@essential-projects/http_contracts": "^2.4.0",
     "@essential-projects/iam_contracts": "^3.6.0",
     "@process-engine/bpmn-elements_contracts": "2.0.0-bc9a859f-b2",
-    "@process-engine/management_api_contracts": "12.1.0-71b8cf24-b2",
+    "@process-engine/management_api_contracts": "14.0.0-alpha.2",
```

Hier das gleiche. Da die Features jetzt doch nicht mit ins Release kommen, ist hier v13.0.0 die korrekte Version.

```suggestion
    "@process-engine/management_api_contracts": "13.0.0-beta.5",
```